File: frontend/ios/ProjectStructure.swift
// ─── iOS Project Structure (Swift + SwiftUI) ───
//
// CarDeal/
// ├── CarDealApp.swift              ← @main entry + TabView
// ├── Info.plist
// ├── DI/
// │   └── Dependencies.swift        ← Service container
// ├── Data/
// │   ├── API/
// │   │   ├── CarDealAPI.swift      ← URLSession / Alamofire
// │   │   └── APIClient.swift       ← Base client + interceptors
// │   ├── Models/
// │   │   └── VehicleReport.swift   ← (already created)
// │   └── Repository/
// │       └── VehicleRepository.swift
// ├── UI/
// │   ├── Theme/
// │   │   ├── AppTheme.swift
// │   │   ├── Colors.swift
// │   │   └── Typography.swift
// │   ├── Screens/
// │   │   ├── Splash/
// │   │   │   └── SplashView.swift
// │   │   ├── Home/
// │   │   │   ├── HomeView.swift
// │   │   │   └── HomeViewModel.swift
// │   │   ├── Report/
// │   │   │   ├── ReportView.swift
// │   │   │   ├── ReportViewModel.swift
// │   │   │   └── Sections/
// │   │   │       ├── VehicleOverviewCard.swift
// │   │   │       ├── SpecsCard.swift
// │   │   │       ├── OwnershipCard.swift
// │   │   │       ├── InspectionCard.swift
// │   │   │       ├── RecallsCard.swift
// │   │   │       ├── TaxCostsCard.swift
// │   │   │       ├── PriceEstimateCard.swift
// │   │   │       ├── RiskCard.swift
// │   │   │       ├── ConditionCard.swift
// │   │   │       ├── SuitabilityCard.swift
// │   │   │       └── AiSummaryCard.swift
// │   │   ├── Plans/
// │   │   │   ├── PlansView.swift
// │   │   │   └── PlansViewModel.swift
// │   │   └── Profile/
// │   │       ├── ProfileView.swift
// │   │       └── ProfileViewModel.swift
// │   ├── Components/
// │   │   ├── SearchBar.swift
// │   │   ├── ReportCard.swift
// │   │   ├── ScoreGauge.swift
// │   │   ├── PriceRangeBar.swift
// │   │   ├── TimelineView.swift
// │   │   ├── AlertBadge.swift
// │   │   ├── CreditBadge.swift
// │   │   └── LoadingOverlay.swift
// │   └── Navigation/
// │       └── TabNavigation.swift
// └── Util/
//     ├── LocaleManager.swift
//     └── DateFormatter.swift

File: frontend/android/ProjectStructure.kt
// ─── Android Project Structure (Kotlin + Jetpack Compose) ───
//
// com.cardeal.app/
// ├── MainActivity.kt
// ├── CarDealApp.kt                ← NavHost + theme
// ├── di/
// │   └── AppModule.kt             ← Hilt dependency injection
// ├── data/
// │   ├── api/
// │   │   ├── CarDealApi.kt        ← Retrofit interface
// │   │   └── ApiClient.kt         ← OkHttp + interceptors
// │   ├── models/
// │   │   └── VehicleReport.kt     ← (already created)
// │   └── repository/
// │       └── VehicleRepository.kt ← Data layer
// ├── ui/
// │   ├── theme/
// │   │   ├── Theme.kt
// │   │   ├── Color.kt
// │   │   └── Type.kt
// │   ├── screens/
// │   │   ├── splash/
// │   │   │   └── SplashScreen.kt
// │   │   ├── home/
// │   │   │   ├── HomeScreen.kt
// │   │   │   └── HomeViewModel.kt
// │   │   ├── report/
// │   │   │   ├── ReportScreen.kt
// │   │   │   ├── ReportViewModel.kt
// │   │   │   └── sections/
// │   │   │       ├── VehicleOverviewCard.kt
// │   │   │       ├── SpecsCard.kt
// │   │   │       ├── OwnershipCard.kt
// │   │   │       ├── InspectionCard.kt
// │   │   │       ├── RecallsCard.kt
// │   │   │       ├── TaxCostsCard.kt
// │   │   │       ├── PriceEstimateCard.kt
// │   │   │       ├── RiskCard.kt
// │   │   │       ├── ConditionCard.kt
// │   │   │       ├── SuitabilityCard.kt
// │   │   │       └── AiSummaryCard.kt
// │   │   ├── plans/
// │   │   │   ├── PlansScreen.kt
// │   │   │   └── PlansViewModel.kt
// │   │   └── profile/
// │   │       ├── ProfileScreen.kt
// │   │       └── ProfileViewModel.kt
// │   ├── components/
// │   │   ├── SearchBar.kt
// │   │   ├── ReportCard.kt
// │   │   ├── ScoreGauge.kt
// │   │   ├── PriceRangeBar.kt
// │   │   ├── TimelineView.kt
// │   │   ├── AlertBadge.kt
// │   │   ├── CreditBadge.kt
// │   │   └── LoadingOverlay.kt
// │   └── navigation/
// │       └── NavGraph.kt
// └── util/
//     ├── LocaleManager.kt
//     └── DateFormatter.kt
